css/selectors/is.json

```
@@ -7,8 +7,20 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/:is",
           "support": {
             "chrome": [
+              {
+                "version_added": "68",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "Experimental Web Platform Features",
+                    "value_to_set": "enabled"
+                  }
+                ],
+                "notes": "Has issues with combinators (see <a href='https://crbug.com/842157'>bug 842157</a>)."
```

While we're here, I thought it might be nice to make it clear that the problem is with combinators in the argument (e.g., `.a :is(.b ~ .c)`), instead of combinators used with `:is` (e.g., `.a ~ :is(.b)`), though it's a bit tough to word it right. What do you think of this?

```suggestion
                "notes": "Combinators in the selector list argument may not match correctly (see <a href='https://crbug.com/842157'>bug 842157</a>)."
```

(If this is a change you'd be open to, then it probably makes sense to change all of these notes in this feature.)